staging/src/k8s.io/kubelet/pkg/apis/dra/v1alpha3/api.pb.go

```diff
@@ -482,7 +482,7 @@ const _ = grpc.SupportPackageIsVersion4
 type NodeClient interface {
 	// NodePrepareResources prepares several ResourceClaims
 	// for use on the node. If an error is returned, the
-	// response is ignored. Failures for individidual claims
+	// response is ignored. Failures for individual claims
 	// can be reported inside NodePrepareResourcesResponse.
 	NodePrepareResources(ctx context.Context, in *NodePrepareResourcesRequest, opts ...grpc.CallOption) (*NodePrepareResourcesResponse, error)
```

Comment:
pb.go files are auto-generated and shouldn't be edited. Please, consider changing .proto files instead.